src/main/java/it/gov/pagopa/pu/sil/service/actualization/ActualizationService.java

```diff
@@ -0,0 +1,58 @@
+package it.gov.pagopa.pu.sil.service.actualization;
+
+import it.gov.pagopa.actualization.legacy.dto.generated.Pagamento;
+import it.gov.pagopa.pu.auth.dto.generated.AccessToken;
+import it.gov.pagopa.pu.auth.dto.generated.UserInfo;
+import it.gov.pagopa.pu.organization.dto.generated.OrgSilService;
+import it.gov.pagopa.pu.sil.connector.actualization.LegacyActualizationService;
+import it.gov.pagopa.pu.sil.connector.organization.service.OrgSilServiceComponent;
+import it.gov.pagopa.pu.sil.dto.generated.AmountUpdatesDTO;
+import it.gov.pagopa.pu.sil.service.AccessTokenService;
+import it.gov.pagopa.pu.sil.service.AuthorizationService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+public class ActualizationService {
+  private final OrgSilServiceComponent orgSilServiceComponent;
+  private final LegacyActualizationService legacyActualizationService;
+  private final AccessTokenService accessTokenService;
+
+  public ActualizationService(OrgSilServiceComponent orgSilServiceComponent,
+                              LegacyActualizationService legacyActualizationService,
+                              AccessTokenService accessTokenService) {
+    this.orgSilServiceComponent = orgSilServiceComponent;
+    this.legacyActualizationService = legacyActualizationService;
+    this.accessTokenService = accessTokenService;
+  }
+
+  public AmountUpdatesDTO actualize(Long orgSilServiceId, String nav,
+                                    UserInfo loggedUser, String accessToken) {
+    OrgSilService orgSilService = orgSilServiceComponent.getOrgSilServiceById(orgSilServiceId, accessToken)
+      .orElseThrow(() -> new IllegalArgumentException("Organization service not found"));
+    AuthorizationService.validateUserForOrganizationId(orgSilService.getOrganizationId(), loggedUser);
+    String orgFiscalCode = AuthorizationService.getOrgFiscalCodeFromUserInfo(loggedUser, orgSilService.getOrganizationId());
+
+    AccessToken actualAccessToken = accessTokenService.getAccessToken(orgSilService, accessToken);
```

Comment:
this could return just the String accessToken...
the AccessToken object is needed inside the Service in order to know about the token expiration... outside of there, we expect to have just a valid token to be used